### ISSUE TITLE
Chat -Green line displays after send attachment/messages offline and come back online.

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#11237]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Launch the app
2. Log in with any account
3. Turn off your internet connection
4. Create a new room
5. Tap on plus button
5. Select Add Attachment
6. Send Attachment
7. Enable the internet connection

## Expected Result:
There is no green line displayed after come back online

## Actual Result:
New line displays after send attachment offline and come back online.

## Workaround:
Unknown

## Platform:

Where is this issue occurring?

- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:** 1.2.5.0

**Reproducible in staging?:** Yes

**Reproducible in production?:** No, can not create room while offline

**Email or phone of affected tester (no customers):** any expensifail account

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://user-images.githubusercontent.com/93399543/191973668-4c1fd8aa-ec33-4dac-beae-84b95db77c75.mp4

**Expensify/Expensify Issue URL:** 

**Issue reported by:** Applause - Internal Team

**Slack conversation:** 

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by sketchydroide** [comment]
is this only on Android, or does this happen in other platforms?
It seems weird to be platform specific

**Comment by sketchydroide** [comment]
@kbecciv ? (sorry forgot to tag you in the previous comment)

**Comment by sketchydroide** [comment]
Also please correct the title, it's a bit confusing as `New line` means a empty line in programing language, aka `return/enter`, did you mean `Green Line`?
thanks :)

**Comment by sketchydroide** [comment]
@marcaaron I think [this change](https://github.com/Expensify/App/blob/261c712e1b8118de955bd31e6cf5b070dd285b3d/src/pages/home/report/ReportActionsList.js#L139-L140) might have been the cause of it? As it's a relative new change, and I think we just added the QA testrail steps yesterday.

This said, I'm not 100% this should be a deploy blocker. Offline first stuff are a new feature, so it makes sense it's not in prod yet. Also this seems to be an edge case.

@kbecciv is this for attachements only, or would it be an issues with messages as wel:?

**Comment by sketchydroide** [comment]
maybe not that change, I think it might be on [componentDidUpdate](https://github.com/Expensify/App/blob/f1dc53487835f580eb7e6c1c69a3082c5848e9ce/src/pages/home/report/ReportActionsView.js#L173-L217)

**Comment by kbecciv** [comment]
@sketchydroide Let me try with messages as well.

**Comment by kbecciv** [comment]
The same issue with message 

https://user-images.githubusercontent.com/93399543/191980433-67e86fb6-3ed9-457d-af6e-048801b0f470.mp4

**Comment by sketchydroide** [comment]
Ok, after some investigation I was able to reproduce the bug in Prod.

Here are the reproduction steps:
1. Launch the app
1. Log in with any account
1. Create a new room
1. Turn off your internet connection
1. write a new comment
1. Enable the internet connection

after a while the green line will appear.


I Don't think this is related to the Add Room Offline functionality. I'm inclined to remove the blocker, as I was able to reproduce something that I think it's the actual issue.

**Comment by sketchydroide** [comment]
aother similar GH opened [here](https://github.com/Expensify/App/issues/11202).

**Comment by marcaaron** [comment]
idk I think it can still be a Daily and added to the Whats App Quality milestone. No reason why someone can't look into this.

**Comment by marcaaron** [comment]
I'm gonna take this off the Unread Indicators milestone and move it to Whats App quality. Looks like tester is using an optimistic report and I have no idea whether unread indicators were taken into consideration for the optimistic report stuff.

**Comment by sketchydroide** [comment]
Ok I'll take a look at this today

**Comment by sketchydroide** [comment]
I can't replicate this on dev, none of the steps previously described, will try again on staging

**Comment by sketchydroide** [comment]
I'm having some trouble checking on this today, I think it's because I'm working from a coworking space today, will chack again on monday from home

**Comment by sketchydroide** [comment]
will try this again today

**Comment by sketchydroide** [comment]
was busy with other things, will take a look today

**Comment by sketchydroide** [comment]
demoting this to weekly, as I don't think this is happening anymore.

@kbecciv can your team take a look to see if it still happens?

**Comment by sketchydroide** [comment]
@kbecciv bump :)

**Comment by kbecciv** [comment]
@sketchydroide Checking, will update you shortly

**Comment by kbecciv** [comment]
@sketchydroide Issue is reproduced with latest build 1.2.16-4

https://user-images.githubusercontent.com/93399543/196197000-be125e5a-3bc3-416e-a42f-7faa4d4b74d4.MP4

**Comment by sketchydroide** [comment]
Thanks, I think it might be part of the process the login part.
let me try again.

**Comment by sketchydroide** [comment]
still haven't had the time to check, busy with the deployment, will check again

**Comment by sketchydroide** [comment]
Checking again this week. once I fix vagrant and php

**Comment by sketchydroide** [comment]
Vagrant fixed, I can look at this again

**Comment by sketchydroide** [comment]
I'm removing myself, I haven't had the time to look at this, and as daily I should be focusing on it and have not been able to

**Comment by jliexpensify** [comment]
Just to confirm, this is what you're referring to @kbecciv ?

![image](https://user-images.githubusercontent.com/30815269/199603163-4a9ee22c-94cc-4a60-9da7-9791521d3ace.png)

if so, I believe that's intended: since it's a new message you're sending, it makes sense to have a green line. I'm on v 1.2.23-0 and am seeing the same behaviour.

**Comment by kbecciv** [comment]
Yes, correct. I am referring to this green line above the attachment.

**Comment by jliexpensify** [comment]
Thanks for confirming - so again, my understanding is that this is intended behaviour, as when you're offline and a message is sent, you shouldn't see them until you get back online, which means they'd show up as "New".

Closing this one out, but please re-open if you feel like this is bug.
